packages/credentials-interface/src/credentials.types.ts

```diff
@@ -1,30 +1,31 @@
 import { ICredentialStatus, ICredentialSubject, IProof } from '@sphereon/pex';
-import { ICredential } from '@sphereon/pex/dist/main/lib/types/SSI.types';
+import { ICredential } from '@sphereon/pex/dist/main/lib/types';
+import { ICredentialContextType } from '@sphereon/pex/dist/main/lib/types/SSI.types';
 import { EIP712Proof } from './eip712proof.types';
 
-export type CredentialSubject = ICredentialSubject;
```

Comment:
We are using this type in ICL for example, let bring it back

Reply:
But it has no value. Why not fix ICL?

Reply:
The main reason is to keep the same name convention. Pex is adding `I` to the beginning of type, we don't

Reply:
Okay